tests/topotests/pim_embedded_rp/test_pim_embedded_rp.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: ISC
+
+# Copyright (c) 2024 Network Education Foundation, Inc. ("NetDEF")
+#                    Rafael Zalamena
+
+import os
+import sys
+import pytest
+from functools import partial
+
+from lib import topotest
+
+# pylint: disable=C0413
+# Import topogen and topotest helpers
+from lib.topogen import Topogen, get_topogen
+from lib.topolog import logger
+
+from lib.pim import McastTesterHelper
+
+"""
+pim_embedded_rp.py: Test PIM embedded RP functionality.
+"""
+
+# Save the Current Working Directory to find configuration files.
+CWD = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(CWD, "../"))
+
+# Required to instantiate the topology builder class.
+pytestmark = [pytest.mark.pim6d]
+
+
+def build_topo(tgen):
```

Comment:
Can you please add a simple topo graph ([example](https://github.com/FRRouting/frr/blob/master/tests/topotests/pim_acl/test_pim_acl.py#L61)). It is priceless when looking at the test in the future when chasing a bug.